### PR TITLE
docs: update link and readme for non sso template

### PR DESCRIPTION
**Pull request by KennethBWSong**

### CODE REVIEW

File: templates/tab/js/non-sso/src/components/sample/AddSSO.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ export function AddSSO(props) {
   const { ssoDocUrl, fluentDocUrl, readmeUrl } = {
     ssoDocUrl: "https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/authentication/auth-aad-sso?tabs=dotnet",
     fluentDocUrl: "https://developer.microsoft.com/en-us/fluentui#/get-started/web",
-    readmeUrl: "https://aka.ms/teamsfx-non-sso-tab-sample-readme", 
+    readmeUrl: "https://aka.ms/teamsfx-add-sso",
```
Comment (hund030) [inline templates/tab/js/non-sso/src/components/sample/AddSSO.jsx:7]: Just curious why there is https but not in README. Make it consistent?

Comment (KennethBWSong) [reply templates/tab/js/non-sso/src/components/sample/AddSSO.jsx:7]: Good catch. Updated in readme, thx！

**Review by hund030 — APPROVED** [review]

**Review by MuyangAmigo — DISMISSED** [review]
LGTM

File: templates/tab/ts/non-sso/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+> Important: Please be advised that access tokens are stored in sessionStorage for you by default. This can make it possible for malicious code in your app (or code pasted into a console on your page) to access APIs at the same privilege level as your client application. Please ensure you only request the minimum necessary scopes from your client application, and perform any sensitive operations from server side code that your client has to authenticate with.
+
+Microsoft Teams supports the ability to run web-based UI inside "custom tabs" that users can install either for just themselves (personal tabs) or within a team or group chat context. Please be advised that mgt-teamsfx-provider library in this app is currently in preview stage, please expect breaking changes in the future release.
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/)
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
+- [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version after 1.55 or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
```
Comment (adashen) [inline templates/tab/ts/non-sso/README.md:11]: 1.55?

File: templates/tab/ts/non-sso/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# How to use this Teams Tab app HelloWorld app
+
+> Important: Please be advised that access tokens are stored in sessionStorage for you by default. This can make it possible for malicious code in your app (or code pasted into a console on your page) to access APIs at the same privilege level as your client application. Please ensure you only request the minimum necessary scopes from your client application, and perform any sensitive operations from server side code that your client has to authenticate with.
```
Comment (adashen) [inline templates/tab/ts/non-sso/README.md:3]: For this warning, normally for the sso token. I sit necessary also have it here?

File: templates/tab/ts/non-sso/README.md (hunk 2 above)
Comment (KennethBWSong) [reply templates/tab/ts/non-sso/README.md]: Updated, thx!

File: templates/tab/ts/non-sso/README.md (hunk 3 above)
Comment (KennethBWSong) [reply templates/tab/ts/non-sso/README.md]: Updated, thx!

**Review by adashen — APPROVED** [review]

**Review by hund030 — DISMISSED** [review]

File: templates/tab/js/non-sso/README.md [hunk 4]
```diff
@@ -63,3 +61,7 @@ Once deployed, you may want to distribute your application to your organization'
 
 - From Visual Studio Code: open the Teams Toolkit and click `Publish to Teams` or open the command palette and select: `Teams: Publish to Teams`.
 - From TeamsFx CLI: run command `teamsfx publish` in your project directory.
+
+## Add Single Sign On feature
+
+Microsoft Teams has provided a mechanism to minimize the number of times users need to enter their sign in credentials and this is called single sign on. Teams Framework (TeamsFx) added support on top of this mechanism to help developers build single sign feature easily. For detail please refer to [Wiki](https://aka.ms/teamsfx-add-sso).
```
Comment (MuyangAmigo) [inline templates/tab/js/non-sso/README.md:67]: Same wording suggestion as non-sso sample, see here: https://github.com/OfficeDev/TeamsFx-Samples/pull/369#discussion_r870431121

**Review by MuyangAmigo — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

**Pull request merged**